packages/paste-website/src/pages/content/get-support.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: Get support
+description: Self-service tips and how to access UX Writer support from Caroline (London, GMT).
+---
+
+import {graphql} from 'gatsby';
+import {Link} from 'gatsby';
+import {Blockquote} from '../../components/Blockquote';
+import {Callout, CalloutTitle, CalloutText} from '../../components/callout';
+import {DoDont, Do, Dont} from '../../components/DoDont';
+import Changelog from '@twilio-paste/icons/CHANGELOG.md';
+
+export const pageQuery = graphql`
+  {
+    mdx(fields: {slug: {eq: "/content/get-support/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<content>
+
+<h1>{props.pageContext.frontmatter.title}</h1>
+
+<p>{props.pageContext.frontmatter.description}</p>
+
+</content>
+
+---
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings}/>
+
+<content>
+
+## Self-service support
+### Content checklist
+
+How do you know when your content is "done" or "good"?  
+
+**The tone is appropriate to the scenario.**
```

Comment:
Do you mind converting these Bolded sub headings to `h4`? I know they may not look the way you'd want, but I would like to keep a fairly strict accessible document outline by using HTML headings as much as possible.

This goes for all pages. I'm not too concerned with those items that appear as boldened list items, but generally if it looks like a heading, and acts like a heading I would _love_ to be using a heading.

Reply:
Great flag, thanks, makes all the sense.  Will fix it up today.